REFACTOR DE PLANEACION_CONSTRUCCION_MF

Mechaz123 opened the pull request:
- Se realiza el refactor de planeacion_construccion_mf, con las funcionalidades migradas.
- Se realiza también el ajuste de los estilos aplicando los colores asociados a planeacion.
- Se realiza el cambio de Endpoints a los micro mid's.

Mechaz123 committed:
feat: udistrital/planeacion_cliente#1006      COMIENZO_REFACTOR_PLANEACION_CONSTRUCCION_MF
- Se comienza con el refactor de planeacion_construccion_mf, se migra el apartado de gestion_usuarios y los demás elementos necesarios para el funcionamiento del mismo.

Mechaz123 committed:
feat: udistrital/planeacion_cliente#1032      REFACTOR_PLANEACION_CONSTRUCCION_MF_PRIMER_MIGRACION

- Se termina de migrar el apartado relacionado al Issue: udistrital/planeacion_cliente#988
- Se migran otras secciones que se encontraron en estos apartados que no se encontraban listados en el PR del Issue.

Mechaz123 committed:
feat: udistrital/planeacion_cliente#1032	    AJUSTE_DE_ESTILOS_Y_CAMBIO_DE_ENDPOINTS
- Se realiza el ajuste de los estilos.
- Se realiza la modificación de los Endpoints para consumir los microservicios desarrollados.

milo19980525 reviewed (commented):
Por favor revisar e incluir: Readme.md, Drone.YML (poner failure.ignore al step de telegram), y en todos los archivos de variables de entorno solo pueden ir aquellas relacionadas con autenticación excepto enviroment.developenemnt.ts (este si lleva las variables de las APIs por intranet)

milo19980525 reviewed (changes requested):
Realizar los ajustes requeridos por favor.

Mechaz123 committed:
feat: udistrital/planeacion_cliente#1050	CORRECCION_DE_ARCHIVOS,ENDPOINTS,ESTILOS_Y_VARIABLES
- Se ajustan los estilos del microfrontend planeacion_construccion_mf, para la seleccion de fechas y otros elementos.
- Se agrega el archivo .drone.yml
- Se modifica el archivo README.md
- Se termina de realizar la corrección de los Endpoints para planes de accion
- Se agregan los archivos de variables de entorno

Mechaz123 commented:
Se envió un commit con los archivos faltantes, y los demás elementos relacionados al issue: https://github.com/udistrital/planeacion_cliente/issues/1050
